acceptance/tests/options/config_file/blocklist.rb

```diff
@@ -0,0 +1,57 @@
+# This test verifies that when a fact group is blocked in the config file
+# the corresponding facts do not resolve.
+test_name "facts can be blocked via a list in the config file" do
+  require 'facter/acceptance/user_fact_utils'
+  extend Facter::Acceptance::UserFactUtils
+
+  agents.each do |agent|
+    step "facts should be blocked when Facter is run from the command line" do
+      step "Agent #{agent}: create config file" do
+        custom_conf_dir = agent.tmpdir("config_dir")
+        config_file = File.join(custom_conf_dir, "facter.conf")
+        create_remote_file(agent, config_file, <<-FILE)
+        facts : { blocklist : [ "file system" ] }
```

Comment:
Would like to add the EC2 blocking here too.  That way we can test the `--debug` output on all platforms, as the `file systems` block group does not attempt resolution on all platforms